.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
+!docker/
```

Comment:
So basically the `.dockerignore` file say:
1. `*`: ignore everything
2. `!docker/` except for the `docker` directory